.github/workflows/post_peek_screenshot.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         if: success()
         env:
           OVERVIEW_IMG_MARKDOWN: ${{ fromJSON(steps.icons_overview_img_step.outputs.markdown_urls)[0] }}
-          DETAILED_IMGS_MARKDOWN: ${{ join(fromJSON(steps.icons_detailed_img_step.outputs.markdown_urls), '\n') }}
+          DETAILED_IMGS_MARKDOWN: ${{ join(fromJSON(steps.icons_detailed_img_step.outputs.markdown_urls), ' ') }}
```

Comment:
Why not just

```suggestion
          DETAILED_IMGS_MARKDOWN: ${{ join(fromJSON(steps.icons_detailed_img_step.outputs.markdown_urls), '') }}
```
?

Reply:
Oh, I was unsure what will happen if two image markdowns are next together. Now that you pointed it out, I just tested it and it seems like there's no issue.